proxygen_cli/test/command_credentials_test.py

```diff
@@ -26,6 +28,19 @@ def get_test_credentials(**kwargs):
 
     return "\n".join(text_format_credentials)
 
+def get_test_client_credentials(**kwargs):
```

Comment:
Ideally we need to cover more scenarios. Should be multiple tests around setting credentials - 
- are the defaults set but we still need to provide username/password
- If we force overwrite username/password can we do that without knowing client id secret
- can we overwrite client id secret if we want to

Reply:
Working on Adding more test cases at the moment.